app/models/miq_report/search.rb

```diff
@@ -93,11 +93,11 @@ def paged_view_search(options = {})
     search_options = options.merge(:class => db, :conditions => conditions, :include_for_find => includes)
     search_options.merge!(:limit => limit, :offset => offset, :order => order) if order
 
-    if options[:parent]
-      targets = get_parent_targets(options[:parent], options[:association] || options[:parent_method])
-    else
-      targets = db_class
-    end
+    targets = if options[:parent] && options[:parent][:class] != "ManageIQ::Providers::Amazon::CloudManager"
```

Comment:
@hstastna feels odd to add this check for a class name.  Is there another way to articulate what you are trying to check here?